server/routes/api/notifications/notifications.ts

```diff
@@ -49,4 +57,121 @@ router.post(
   handleUnsubscribe
 );
 
+router.post(
+  "notifications.list",
+  auth(),
+  pagination(),
+  validate(T.NotificationsListSchema),
+  transaction(),
+  async (ctx: APIContext<T.NotificationsListReq>) => {
+    const eventType = ctx.input.body.eventType ?? ctx.input.query.eventType;
+    const archived = ctx.input.body.archived ?? ctx.input.query.archived;
```

Comment:
```suggestion
    const { eventType, archived } = ctx.input.body;
```

I don't think we need to support query params here, it's a POST endpoint